packages/eas-cli/src/credentials/context.ts

```diff
@@ -112,3 +114,12 @@ export class CredentialsContext {
     this.shouldAskAuthenticateAppStore = false;
   }
 }
+
+export function hasProjectContext(projectDir: string): boolean {
```

Comment:
This logic seems to be duplicated. Can you please refactor `CredentialsContext#hasProjectContext` so it reuses the function?

Reply:
`CredentialsContext#hasProjectContext` is a memoized version of `hasProjectContext`. `hasProjectContext` tries to read the expo config every time, which can be expensive . 

But I agree it is not great to have duplicate method names. To avoid confusion, I've changed this around.